cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -392,15 +392,13 @@ def process_soc_data(
         F.coalesce(F.col("resolved_soc_code"), F.col("standard_occupational_classification_code")),
     ).drop("resolved_soc_code")
 
+    # remove any duplicate
     soc_lookup_df = soc_lookup_df.withColumn("FILTER_CONDITION", F.count("*").over(window) > 1)
```

Comment:
This will remove all cases of complete duplicates, but we would want to keep one copy if the duplicates were identical on all columns. This logic belongs in the start of `transform_cis_soc_data` so that it's covered by a test too